### DISCUSSION TITLE
Ability to get alerts when the phone disconnects

### DISCUSSION THREAD

**Discussion by DavisNT** [InfiniTimeOrg/InfiniTime#2095] [Ideas]
I think optional (controlled by a setting) alerts when Bluetooth disconnects could be very useful in some situations.

**Comment by tituscmd** [comment]
I’d suggest adding an option in the settings for the watch to wait a few seconds (like 30 seconds or 1 minute) before notifying the user about a lost connection. This way, it can avoid alerting them about brief or temporary disconnections.

**Comment by cmetek** [comment]
as forgetful person I agree, and upvote. Also perhaps a button to ring my phone when it is connected so I can find it easier, (like if my phone fell behind the couch, under my office desk, is in my backpack etc.
